### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by aljones15**
1. fixes all tests to `100%` success rate
2. Upgrades `@digitalbazaar/zcapld: ^5.0.0` & `bedrock-ledger-context: ^20.0.0`
3. Removes deprecated option `verificationMethod` from `ed25519-2020` keys
4. Adds a zcap context to each operation before signing
5. Signs each operation with a `CapabilityInvocation` purpose even when doing `ledgerConfig` operations.

### CODE REVIEW

File: test/mocha/35-operations-api.js
Comment (dlongley) [inline test/mocha/35-operations-api.js:87]: If the test context is going to be used here then `value` needs to be defined in that context.

File: test/mocha/35-operations-api.js
Comment (dlongley) [inline test/mocha/35-operations-api.js:112]: Swap order.

File: test/mocha/helpers.js
Comment (dlongley) [inline test/mocha/helpers.js:192]:
```suggestion
      // doing this sort of optionality would be dangerous in a real system
      // (you want to be very deliberate about your invocation target)
      // but this is just a helper for unit tests so it is considered acceptable
      invocationTarget: invocationTarget || doc.ledger || doc.record.id,
```

**Review by dlongley — CHANGES_REQUESTED** [review]
Thanks! A few comments to address.

File: test/mocha/35-operations-api.js
Comment (dlongley) [reply test/mocha/35-operations-api.js:112]: Oh, nvm, this is a negative test.

File: test/mocha/35-operations-api.js
Comment (aljones15) [reply test/mocha/35-operations-api.js]: Changed value to name: https://github.com/digitalbazaar/bedrock-ledger-node/commit/f59451ec5828d97289c38c2eec252b3f842ed8bc

**Review by dlongley — APPROVED** [review]
Thanks!

**Comment by aljones15** [comment]
This PR is passing a multi-node test using `did-bench-cli` so I'm ok with a release. Waiting on @gannan08 for second approval and release.

File: test/mocha/35-operations-api.js
Comment (gannan08) [inline test/mocha/35-operations-api.js:87]: @aljones15 Can you expand on the property name change?

File: test/mocha/helpers.js
Comment (gannan08) [inline test/mocha/helpers.js:176]:
```suggestion
  // if the context is an array and the ZCAP context isn't included, add it
```

File: test/mocha/helpers.js
Comment (gannan08) [inline test/mocha/helpers.js:180]:
```suggestion
  // if the context isn't an array, then make it an array with the ZCAP context
```

**Review by gannan08 — CHANGES_REQUESTED** [review]

File: test/mocha/35-operations-api.js
Comment (aljones15) [reply test/mocha/35-operations-api.js:87]: We're using this as `name` is a term in the `context` while `value` is not in the `context`.

File: test/mocha/helpers.js
Comment (aljones15) [reply test/mocha/helpers.js:176]: this might add a linting error for line length. The next line continues the sentence.

File: test/mocha/35-operations-api.js
Comment (gannan08) [reply test/mocha/35-operations-api.js:87]: Great, sounds good.

**Review by gannan08 — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**
